app/assets/sass/application.scss

```diff
@@ -76,6 +76,10 @@ table.govuk-table {
   min-width: 40%;
 }
 
+.govuk-table__row--versions:hover {
+  background-color: unset !important;
```

Comment:
Uh-oh, `!important`, the sign of CSS distress. 😆 
I wouldn't worry too hard about it in the prototype, though.